Changes

```diff
@@ -209,6 +213,10 @@ OCaml 4.14.0
   WSADuplicateSocket on sockets instead of DuplicateHandle.
   (Antonin Décimo, review by Xavier Leroy and Nicolás Ojeda Bär)
 
+- #10???: Prefix all C functions in the Unix library with `unix_` except for the
```

Comment:
What about using `caml_unix_` instead?

Reply:
All the other functions already used `unix_`, so it makes it consistent with them. I'm not against that change either, though :)